inspirehep/records/api/base.py

```diff
@@ -486,3 +486,7 @@ def serialize_for_es(self):
     def get_value(self, field, default=None):
         """Method which makes ``get_value`` more intuitive"""
         return get_value(self, field, default)
+
+    def get_modified_references(self):
+        """Method which is implemented only for LiteratureRecord. For other records, it will return empty list"""
```

Comment:
Having a condition like `if literature record` on the caller seems more easier to understand to me than this.

Reply:
yes. We had a similar check before but then the method was used somewhere else as well (in the migrator) and the check was missing and it was blowing up. So we decided that it is safer to have it like this so that in case someone forgets to add the check, it wouldn't blow up anymore.

Reply:
well it was blowing, because it was supposed to blow up. If we keep doing this base class would end up full of irrelevant crap.

The problem here is that we didn't realized that was happening, not that it was "unsafe" to put the check there.